docs/how-to-guides/session-customizing.rst

```diff
@@ -31,57 +31,83 @@ done automatically for Python and R projects if you add the packages you want
 to ``requirements.txt`` and ``install.R`` respectively.
 
 
+.. _renku_project_config:
+
 Renku project configurations
 ----------------------------
 
 When starting a new Session, most of the options can be manually
 changed by the user. Depending on the specific RenkuLab deployment, you can select
-more RAM, a higher CPU quota, etc.
+more RAM, more storage space, a higher CPU quota, etc.
 
 Your project may even include a package with an advanced UI (like
 `Streamlit <https://renku.discourse.group/t/how-to-deploy-streamlit-in-renku/169>`_)
 and you probably want to choose it as default.
 
-It's possible to set a default value for all these options using the project
-configurations stored in the ``.renku/renku.ini`` file.
-Once you do that, each time a user tries to start a new environment, those options will
-be pre-selected.
+It's possible to set a default value for all these options, either using the
+`renku config command`_ locally or in a session, or directly on RenkuLab project
+page.
+
+To do that, you can go into the project `Settings` section and click on the
+`Sessions` tab. The interface shows the available project settings. Mind that you
+can only select values valid to the specific RenkuLab platform. You can still use
+the CLI to set any custom value, but that may not work as expected.
+
+.. image:: ../../_static/images/project-session-settings.png
```

Comment:
this image has a black border on the left - can we trim it just a bit so the black border is gone?